packages/destination-actions/src/destinations/optimizely-feature-experimentation-actions/trackEvent/index.ts

```
@@ -108,6 +108,9 @@ const action: ActionDefinition<Settings, Payload> = {
   },
   perform: async (request, { settings, payload }) => {
     const result = await request<ProjectConfig>(settings.dataFileUrl)
+    if (!result.data && typeof result.content === 'string') {
```

nit: result.content's type is a string. Should we just check if result.content exists?

Also, what should we do if result.content and result.data both are missing?

---

I don't think there will be condition where these two will be missing other than if there will be error. But we can throw a customized error message in that condition.